include/alpaka/vec/Vec.hpp

```diff
@@ -46,13 +46,6 @@
 #include <type_traits>                      // std::enable_if, std::decay, std::is_same
 #include <algorithm>                        // std::min, std::max, std::min_element, std::max_element
 
-// Some compilers do not support the out of class versions:
-// - the nvcc 7.0 CUDA compiler
```

Comment:
Testing showed, that this work around is required even for CUDA 7.5 and CUDA 8.0...